push.go

```diff
@@ -53,16 +53,53 @@ func (cmd *PushCommand) Run() error {
 type Sources []*Source
 
 type Source struct {
-	File          string      `yaml:"file,omitempty"`
-	ProjectID     string      `yaml:"project_id,omitempty"`
-	AccessToken   string      `yaml:"access_token,omitempty"`
-	FileFormat    string      `yaml:"file_format,omitempty"`
-	Params        *phraseapp.UploadParams `yaml:"params"`
+	File          string
+	ProjectID     string
+	AccessToken   string
+	FileFormat    string
+	Params        *phraseapp.UploadParams
 
 	RemoteLocales []*phraseapp.Locale
 	Extension     string
 }
 
+func (src *Source) UnmarshalYAML(unmarshal func(interface{}) error) error {
```

Comment:
* the receiver is different
* there are some subtle differences with regard to special variables supported by target